File: Java/Driver.java
package Java;

public class Driver extends Account {
    public Driver(String name, String document, String email, String password) {
        super(name, document);
    }

    void PrintDataDriver() {
        System.out.println(
                "Your name: " + name + " And document: " + document + " Email: " + email + " Password: " + password);
    }
}

File: JS/Driver.js
class Driver extends Account {
  constructor(name, document, email, password) {
    super(name, document, email, password);
  }

  PrintDataDriver = () => {
    console.log("Nombre:" + this.name);
    console.log("Documento:" + this.document);
    console.log("Email:" + this.email);
    console.log("Password:" + this.password);
  };
}
